roles/test_operator/tasks/stages.yml

```diff
@@ -0,0 +1,33 @@
+---
+- name: "Run stage {{ _stage_vars.name }}"
+  ansible.builtin.debug:
+    msg: "Run stage {{ _stage_vars.name }}"
+
+- name: "Call pre stage hooks for test operator stage {{ _stage_vars.name }}"
+  when:
+    - not cifmw_test_operator_dry_run | bool
+    - _stage_vars.pre_test_stage_hooks is defined
+  vars:
+    step: pre_test_hooks
+    hooks: "{{ _stage_vars.pre_test_stage_hooks }}"
+  ansible.builtin.import_role:
+    name: run_hook
+
+- name: Include stage var file
+  when: _stage_vars.test_file is defined
+  ansible.builtin.include_vars:
+    file: "{{ _stage_vars.test_file | default(ansible_user_dir ~ '/ci-framework-data/parameters/reproducer-variables.yml') }}"
```

Comment:
Do we need the `default()` here since this task will be executed only when `test_file` is defined?

Reply:
correct this is a change I added later will remove